src/Model.jl

```diff
@@ -42,11 +43,11 @@ struct Model{T <: Real}
     term_types::Vector
 
     # list of symmetries of the model
-    symops::Vector{SymOp}
+    symmetries::Vector{SymOp}
```

Comment:
Hm I think I liked symops better here. Symmetries is a bit vague, this is symmetry operations (symops), which have a well-defined meaning as a pair or rotation and translation

Reply:
I did that for consistency. This way the parameter to `Model` and the field has the same name. For the parameter I find `symmetries=false` more easy to understand than `symops=false`. I think it's ok this way since we are consistent everywhere in the code: `symmetries` is always the symmetry operations.

Reply:
OK. I still feel `symmetries` is too vague, but one look at the `Model` structure reveals it to be a vector of symop, which is documentation in itself. Agreed then!